script/brew-publish

```diff
@@ -46,7 +46,7 @@ rm -f "${formula}.bak"
 git commit -m "${brew_name} ${version#v}" -- "$formula"
 # hackish way of getting the git remote name for user's fork
 fork_remote="$(hub fork 2>&1 | grep -oE 'remote:? \S+' | tail -1 | awk '{print $2}')"
-git push -u "$fork_remote" "$branch"
+git push -u "${fork_remote}" HEAD
```

Comment:
Yeah they are stylistic because it's easier to read where string interpolation begins and ends. It's never neccessary when just quoting a single variable, though.